server/authorizer/authorizer_test.go

```diff
@@ -455,6 +457,64 @@ func TestMakeInputWithBody(t *testing.T) {
 
 }
 
+func TestInterQueryCache(t *testing.T) {
+
+	count := 0
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		count += 1
```

Comment:
[nit]
```suggestion
		count++
```

Reply:
Hehe, context switching from Python does that I guess :D